components/canvas/Canvas.tsx

```diff
@@ -24,3 +20,26 @@ function Canvas() {
   );
 }
 export default Canvas;
+
+const albert = { x: 1360, y: 250, width: 50, height: 10 };
+
+function moveAlbert(context) {
+  const image = new Image();
+  image.src = "/Albert.svg";
+  image.onload = () => {
+    context.drawImage(image, albert.x, albert.y);
+  };
+  context.clearRect(0, 0, albert.width, albert.height);
```

Comment:
Hat es einen bestimmten Grund, dass du zwei clearRect benutzt? Würde der gleiche Effekt mit einer clearRect in den Zeilen über deinem drawImage erziehlt werden?

Im Moment sieht es so aus als würdest du Albert malen, dann ihn "clearen", ihn bewegen - ohne ihn nochmal zu malen - und dann das celearte Rect nochmal clearen. But maybe I am missing something here 😅